lace_ground.py

```diff
@@ -172,42 +172,36 @@ def __init__(self):
         inkex.Effect.__init__(self)
 
         # file
-        self.arg_parser.add_argument('-f', '--file', action='store', type=str, dest='file', help='File containing lace ground description')
+        self.arg_parser.add_argument('-f', '--file',
+                                     type=str,
+                                     dest='file',
+                                     help='File containing lace ground description')

```

Comment:
Why is this help lacking in the circular version?

Reply:
Not intentional.  I have added this help.